test/cmdlineTests.sh

```diff
@@ -52,12 +54,26 @@ do
             shift
             ;;
         *)
-            selected_tests+=("$1")
+            matching_tests=$(find . -mindepth 1 -maxdepth 1 -type d -name "$1" | cut --characters 3- | sort)
```

Comment:
I might be wrong, but I thought this kind of `find` call for `-name` does only an exact match. Is that intended here?

Reply:
It's intended. It does not do a regex match but still supports wildcards.

Actually the argument to `-name` should always be quoted because otherwise shell will expand `*` and `?`. Though it usually kinda works as long as you are only searching the current directory.

Reply:
Ah right, you can just pass a whiledcard there. good then :)